.idea/misc.xml

```diff
@@ -46,7 +46,11 @@
       </value>
     </option>
   </component>
+<<<<<<< HEAD
```

Comment:
Those merge annotation should be deleted. I'm though clueless as to why this compiles at all

Reply:
Done